types/ahocorasick/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for ahocorasick 1.0.2
+// Project: https://github.com/BrunoRB/ahocorasick
+
+export default class AhoCorasick {
```

Comment:
```suggestion
export class AhoCorasick {
```

Reply:
@codershiba Shouldn't it be correctly written as follows?

```diff typescript
- export default class AhoCorasick {
+ declare class AhoCorasick {
  // ...
  }
+ export = AhoCorasick;
```